aries-site/src/themes/aries.js

```diff
@@ -0,0 +1,110 @@
+import { hpe } from 'grommet-theme-hpe';
+import { deepMerge } from 'grommet/utils';
+
+export const aries = deepMerge(hpe, {
+  name: 'HPE 2',
+  rounding: 6,
+  spacing: 24,
+  defaultMode: 'dark',
+  global: {
+    colors: {
+      brand: 'green',
+      background: {
+        dark: '#1A1F2B',
+        light: '#FFFFFF',
+      },
+      'background-strong': {
+        dark: '#000000',
+        light: '#FFFFFF',
+      },
+      'background-weak': {
+        dark: '#44444480',
+        light: '#E8E8E880',
+      },
+      'background-xweak': {
+        dark: '#66666699',
+        light: '#CCCCCC90',
+      },
+      text: {
+        dark: '#EEEEEE',
+        light: '#333333',
+      },
+      'text-strong': {
+        dark: '#FFFFFF',
+        light: '#000000',
+      },
+      'text-weak': {
+        dark: '#CCCCCC',
+        light: '#444444',
+      },
+      'text-xweak': {
+        dark: '#999999',
+        light: '#666666',
+      },
+      border: 'background-xweak',
+      control: 'brand',
+      'active-background': 'background-weak',
+      'active-text': 'text-strong',
+      'selected-background': 'brand',
+      'selected-text': '',
```

Comment:
This looks suspicious, what it be preferred to remove it?

Reply:
Good catch. Setting as undefined for now.